WhoIsThat/WhoIsThat/Connections/RestService.cs

```diff
@@ -15,82 +16,118 @@ namespace WhoIsThat.Connections
 {
     public class RestService : IRestService
     {
-        public IHttpClientHandler _httpHandler { get; set; }
+        public HttpClient HttpClient { get; set; }
         public RestService()
         {
-            _httpHandler = new HttpClientHandler();
+            HttpClient = new HttpClient();;
         }
 
-        /// <summary>
-        /// Calls ImageObjectElementController in backend and returns list of image objects
-        /// </summary>
-        /// <returns>List of ImageObject instances</returns>
+        /// <inheritdoc/>
         public async Task<List<ImageObject>> GetImageObjects()
         {
             try
             {
-                string restUrl = "https://teststorageserver.azurewebsites.net/api/images/all";
+                const string restUrl = "https://teststorageserver.azurewebsites.net/api/images/all";
                 var uri = new Uri(string.Format(restUrl, string.Empty));
 
-                var response = await _httpHandler.Get(uri);
-                if (response.IsSuccessStatusCode)
-                {
-                    var content = await response.Content.ReadAsStringAsync();
-                    return JsonConvert.DeserializeObject<List<ImageObject>>(content);
-                }
-                else
-                {
-                    throw new Exception("Something went wrong: " + response.StatusCode);
-                }
+                var response = await HttpClient.GetAsync(uri);
+                if (!response.IsSuccessStatusCode) throw new Exception();
```

Comment:
This is a very minor thing, but you should always wrap if/else in {}, even if its one line

Reply:
I thought that too, but then JetBrains editor suggested it in that way, and I always though that editor knows better :D